go/llvm-apple/Makefile

```
@@ -3,13 +3,20 @@ include ../Makefile.common
 SUFFIX         := $(shell basename $(CURDIR))
 TAG 		   := $(REPOSITORY)/$(NAME):$(SUFFIX)$(TAG_EXTENSION)
 
+ifeq ($(DOCKER_MULTIARCH),1)
+DOCKER_CMD := $(SELF_DIR)/../.ci/scripts/buildx.sh
+
+push:
+	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```

```suggestion
```suggestion
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
atomic-push:
	@echo "Already done by buildx (.ci/scripts/buildx.sh)."
```
```